### ISSUE TITLE
Project modification Awtrix style

### ISSUE THREAD

**Issue by dcjona** [PlanetaryMotion/pingPongBallClock#53]
Hi 
Since i found tout project i fell in love with it.
So i Just want to share the second clock i'm making a little différent 
![IMG_20200323_124748](https://user-images.githubusercontent.com/34706426/77467252-f3129b80-6e0b-11ea-880f-e7251eb8994a.jpg)
![IMG_20200323_124350](https://user-images.githubusercontent.com/34706426/77467267-fad24000-6e0b-11ea-8232-ccd1a7a319e2.jpg)

![IMG_20200323_125451](https://user-images.githubusercontent.com/34706426/77467315-14738780-6e0c-11ea-9193-534d5af6ebb5.jpg)
![IMG_20200323_144556](https://user-images.githubusercontent.com/34706426/77467341-1c332c00-6e0c-11ea-9576-d81a6666e063.jpg)

**Comment by dcjona** [comment]
![JPEG_20200324_142338](https://user-images.githubusercontent.com/34706426/77467517-6f0ce380-6e0c-11ea-849a-0417354a22f7.jpg)
![JPEG_20200324_142451](https://user-images.githubusercontent.com/34706426/77467522-72a06a80-6e0c-11ea-8bc9-e12e88e0d3b4.jpg)
Maybee you could help me to modify order to make the number correctly ? You could see the number 1 like the ping pong clock so in wrong shape for this project

**Comment by lancewilhelm** [comment]
Wow this is very cool @dcjona! 

So the process of getting the numbering right should be the same that we did for your other issue. If I am not mistaken, this is just an ordering issue based on how you chose to wire your LEDs, right? Go ahead and follow what we did in the first thread that I worked with you and see if you can figure out the order. 

Thanks!

**Comment by dcjona** [comment]
i make same order of your project and didn't modified anything. maybee it's made for pingpong ball and not for squared? will check again :)

**Comment by lancewilhelm** [comment]
Ahhh, I am actually tracking this a bit better now. I understand how your square project is ordered. 

Yes, when we do the ping pong ball clock we stagger the LED lights on the balls. It looks like in your square project your LEDs are straight up and down. This will definitely change how the layout is displayed. However, you just need to perform the same process that we did for your first layout issue. You just need to write a diagram and then map the LED numbers for each location. Then create the row numbers from that.

**Comment by dcjona** [comment]
Number ans rows are the same that the one showed on the readme no? I follow the same order for thé led diagram

**Comment by lancewilhelm** [comment]
It looks like your device only has 7 rows so you will have to configure it with that in mind.

**Comment by dcjona** [comment]
Ping pong ball has also 7 rows not right?

**Comment by lancewilhelm** [comment]
Yes, but I encourage you to think of your clock as something new and different and determine the row numbering, order, count, on your own. This will allow you to fully customize it and have it work the first time.

**Comment by dcjona** [comment]
Maybee the 999 number have to be in différent order too?

**Comment by lancewilhelm** [comment]
They may. I cannot say for certain though. You will need to experiment with
it.
